README.md

```diff
@@ -6,33 +6,66 @@
 [![Discourse status](https://img.shields.io/discourse/https/meta.discourse.org/status.svg?style=for-the-badge)](https://forums.habitat.sh)
```

Comment:
Little thing I just noticed, which it'd be nice to fix since you're in there: Could we change this one to `style=flat` to match the others?

Reply:
That will be great! I noticed that as well but didn't know how to adjust it. Thanks, Chris!